src/amp-story-embed-manager.js

```diff
@@ -14,57 +14,76 @@
  * limitations under the License.
  */
 
-import {setStyle} from '../src/style';
+import {isExperimentOn} from './experiments';
+import {setStyle} from './style';
 
 /** @enum {string} */
 const LoadStateClass = {
-  LOADING: 'loading',
-  LOADED: 'loaded',
-  ERROR: 'error',
+  LOADING: 'i-amphtml-story-embed-loading',
+  LOADED: 'i-amphtml-story-embed-loaded',
+  ERROR: 'i-amphtml-story-embed-error',
 };
 
 /** @const {string} */
 const CSS = `
   :host { all: initial; display: block; border-radius: 0 !important; width: 405px; height: 720px; overflow: auto; }
   .story { height: 100%; width: 100%; flex: 0 0 100%; border: 0; opacity: 0; transition: opacity 500ms ease; }
   main { display: flex; flex-direction: row; height: 100%; }
-  .loaded iframe { opacity: 1; }
+  .i-amphtml-story-embed-loaded iframe { opacity: 1; }
+  iframe[src=""] { display: none; }
 `;
 
+/** @const {boolean} */
+const isAMP = self.AMP && self.AMP.ampdoc;
+
 /**
  * Note that this is a vanilla JavaScript class and should not depend on AMP
  * services, as v0.js is not expected to be loaded in this context.
  */
-export class AmpStoryEmbed {
+export class AmpStoryEmbedManager {
   /**
+   * @param {!Window} win
    * @param {!Document} doc
-   * @param {!Element} element
+   * @param {!Element} hostEl
    * @constructor
    */
-  constructor(doc, element) {
+  constructor(win, doc, hostEl) {
     console./*OK*/ assert(
-      element.childElementCount > 0,
+      hostEl.childElementCount > 0,
       'Missing configuration.'
     );
 
     /** @private {!Element} */
-    this.element_ = element;
+    this.hostEl_ = hostEl;
 
     /** @private {!Document} */
     this.doc_ = doc;
 
-    /** @private {!Array<!HTMLAnchorElement>} */
-    this.stories_ = Array.prototype.slice.call(element.querySelectorAll('a'));
+    /** @private {?Array<!HTMLAnchorElement>} */
+    this.stories_;
```

Comment:
You need to set these to null. As is, these don't actually create the field on the object, which causes polymorphism.